src/modules/esl-a11y-group/core/esl-a11y-group.ts

```diff
@@ -0,0 +1,97 @@
+import {ExportNs} from '../../esl-utils/environment/export-ns';
+import {ESLBaseElement, attr, boolAttr} from '../../esl-base-element/core';
+import {bind} from '../../esl-utils/decorators/bind';
+import {ARROW_DOWN, ARROW_LEFT, ARROW_RIGHT, ARROW_UP} from '../../esl-utils/dom/keys';
+import {TraversingQuery} from '../../esl-traversing-query/core/esl-traversing-query';
+
+export type GroupTarget = 'next' | 'prev' | 'current';
+
+@ExportNs('A11yGroup')
+export class ESLA11yGroup extends ESLBaseElement {
+  public static is = 'esl-a11y-group';
+
+  @attr({defaultValue: '::child'}) public targets: string;
+  @boolAttr({}) public clickOnActive: boolean;
+
+  /**
+   * @returns {HTMLElement} parent element of this
+   */
+  public get $parent(): HTMLElement {
```

Comment:
Minor: mb we can choose something more abstract (root, scope)
It gives us more understanding for what purpose it is used and leaves us room for maneuver in case we change the strategy for that component.

Reply:
Good idea! Thanks!